Add sonar to project for quality checks (#452)
exasol/python-toolbox: doc/user_guide/getting_started.rst

8. Set up for Sonar
+++++++++++++++++++

> We use SonarQube Cloud to analyze, visualize, & track linting, security, & coverage. In

Reviewer: ```suggestion
PTB supports using SonarQube Cloud to analyze, visualize, & track linting, security, & coverage. In
```

order to properly set it up, you'll need to do the following instructions for each **public** project.

> At this time, we do not currently support setting up SonarQube for a **private** project.

Reviewer: ```suggestion
At this time, PTB currently does not support setting up SonarQube for a **private** project.
```
